File: GravStark.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main() {
    string name = "@GravStark";
    string nationality = "Italian";
    int age = 18;
    string city = "Modena";

    vector<string> interests = {"Programming", "VideoGames", "Music", "Healthcare"};
    vector<string> languages = {"Italian", "English", "Finnish"};
    vector<string> programmingLanguages = {"C", "C++", "C#", "Batch", "Assembly", "Java", "Html", "CSS", "JavaScript"};

    cout << "Hi, I'm " << name << ", an " << nationality << " " << age << " y/o student from " << city << "." << endl;
    cout << "✨ Interests:" << endl;
    for (const string& interest : interests) {
        cout << "- " << interest << endl;
    }
    cout << "✨ Languages:" << endl;
    for (const string& language : languages) {
        cout << "- " << language << endl;
    }
    cout << "✨ Programming languages:" << endl;
    for (const string& programmingLanguage : programmingLanguages) {
        cout << "- " << programmingLanguage << endl;
    }

    return 0;
}

File: GravStark.py
def main():
    name = "@GravStark"
    nationality = "Italian"
    age = 18
    city = "Modena"

    interests = ["Programming", "VideoGames", "Music", "Healthcare"]
    languages = ["Italian", "English", "Finnish"]
    programming_languages = ["C", "C++", "C#", "Batch", "Assembly", "Java", "Html", "CSS", "JavaScript"]

    print(f"Hi, I'm {name}, an {nationality} {age} y/o student from {city}.")
    print("✨ Interests:")
    for interest in interests:
        print(f"- {interest}")
    print("✨ Languages:")
    for language in languages:
        print(f"- {language}")
    print("✨ Programming languages:")
    for programming_language in programming_languages:
        print(f"- {programming_language}")

if __name__ == "__main__":
    main()
